website/pages/gamejam_en.md

```diff
@@ -1,6 +1,22 @@
 <img src="/static/img/logga.png" alt="LiU Game Jam" id="gamejam-logo">
-<h1>MicroJam 2020</h1>
-Get ready for our upcoming event [Microjam](/microjam/en/)!
+<h1>Fall Game Jam 2020</h1>
+Get ready for our upcoming event Fall Game Jam!
+This year's edition of Fall Game Jam is held 13th - 15th November.
+Because COVID-19 still is wreaking havoc in the world, this jam will be held remotely on our [Discord server](https://discord.gg/YXV6dpc). Also check in at our [Facebook event](https://www.facebook.com/events/4918751154809003/).
```

Comment:
```suggestion
Because COVID-19 still is wreaking havoc in the world, this jam will be held remotely on our [Discord server](https://discord.gg/YXV6dpc). Remember to checkout our [Facebook event](https://www.facebook.com/events/4918751154809003/).
```

Reply:
check out